client/state/sites/media-storage/test/selectors.js

```diff
@@ -57,4 +58,44 @@ describe( 'selectors', () => {
 			expect( isRequestingMediaStorage( state, 'not-defined' ) ).to.eql( false );
 		} );
 	} );
+	describe( '#isOverMediaLimit()', () => {
+		it( 'should return true if a site is over storage limits', () => {
+			const state = deepFreeze( {
+				sites: {
+					mediaStorage: {
+						items: {
+							2916284: {
+								max_storage_bytes: 3221225472,
+								storage_used_bytes: 3221225900
+							},
+							77203074: {
+								max_storage_bytes: 3221225472,
+								storage_used_bytes: 323506
+							}
+						}
+					}
+				}
+			} );
+
+			expect( isOverMediaLimit( state, 2916284 ) ).to.eql( true );
+			expect( isOverMediaLimit( state, 77203074 ) ).to.eql( false );
+		} );
+		it( 'should return false if a site is unlimited or missing', () => {
+			const state = deepFreeze( {
```

Comment:
Not that it doesn't add a bit more confidence, but do we really need `deepFreeze` in a selector test?